### ISSUE TITLE
Accessibility support and screen readers 

### ISSUE THREAD

**Issue by MesterPerfect** [manifestinteractive/weather-bar-app#39]
#### Overview:
I cannot use your Program with screen readers for blind users 

#### Acceptance Criteria: 
I downloaded Weather Bar
After that, I installed it on windows 10 and windows 11
But I can't navigate the program with a screen reader for blind users 

#### Steps to Duplicate _( required for bug reports )_:

All I can say is that Weather Bar needs accessibility support 
And since it is built in languages like html, it can easily support accessibility 

#### System Info _( required for bug reports )_:

Operating system: windows 10 and windows 11 
Browser version : Version 99.0.4844.51 (Official Build) (64-bit)

**Comment by manifestinteractive** [comment]
This was never designed for that as the entire app was meant as a visual experiment ( animated backgrounds based on weather and time of day, etc ) so I had not considered that others might still choose to install the app for purely functional reasons.  A pretty straight forward fix, I just did not invest time in it since it was a visual experiment.

**Comment by manifestinteractive** [comment]
**Reopening Issue.**

> Previously closed as I was planning on closing down the app for being too expensive for me to run, but going to see if [Community Support](https://github.com/manifestinteractive/weather-bar-app#project-support) via donations can at least pay the server bills I have to keep this free app running for folks.
